doc/Language/regexes.pod6

```diff
@@ -1020,21 +1020,21 @@ anchoring the regex match to that position.
 
 The C<^> anchor only matches at the start of the string:
 
-    say so 'properly' ~~ /  perl/;    # OUTPUT: «True␤»
-    say so 'properly' ~~ /^ perl/;    # OUTPUT: «False␤»
-    say so 'perly'    ~~ /^ perl/;    # OUTPUT: «True␤»
-    say so 'perl'     ~~ /^ perl/;    # OUTPUT: «True␤»
+    say so 'prorakuy' ~~ /  raku/;    # OUTPUT: «True␤»
```

Comment:
This is not good... Again, probably better to PR just a file or a bunch of files... You might have changed some occurrence of Perl